packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -3207,6 +3300,19 @@ function beginWork(
       }
       break;
     }
+    case Chunk: {
+      if (enableChunksAPI) {
+        const chunk = workInProgress.type;
+        const props = workInProgress.pendingProps;
+        return updateChunk(
+          current,
+          workInProgress,
+          chunk,
+          props,
+          renderExpirationTime,
+        );
+      }
+    }
```

Comment:
Kind of inconsistent we don't throw for scopes/fundamental when their flags are off. Maybe add a `break` here to be consistent? Or fix the other ones to throw by removing their breaks.

Reply:
We do throw. If you look, this has the same semantics because the default case isn't a `default` but outside the block. `break` means default.

This should allow us to dead code eliminate the case statements since they all fall into the default case.

However, we're kind of inconsistent with where we do this. Pattern.